explore.py

```diff
@@ -21,6 +21,11 @@ def main(args):
     print("The most visit times for a customer in train set is: ",
           explore_utils.find_most_visit(data))
 
+    #   Mean number of visits for socially engaged customers
+
+    print("The mean visit times for socially engaged customers is: ",
```

Comment:
The description here doesn't match the PR description. Assuming your function finds the mean revenue of socially active customers, then the description should be "The mean ~~visit times~~ **revenue** for socially engaged customers"